src/apps/filters/h3.c

```diff
@@ -35,47 +35,51 @@
 #include "h3Index.h"
 #include "utility.h"
 
+#define PARSE()                                                             \
+    if (parseArgs(argc, argv, sizeof(args) / sizeof(Arg *), args, &helpArg, \
+                  args[0]->helpText)) {                                     \
+        return E_SUCCESS;                                                   \
+    }
```

Comment:
Nit: Do we have a `.h` header file for this? Should we? If we did, should this macro go there?

Reply:
We kinda do, but it's shared across many of these binaries while this is only useful here.

On top of that, if I put it in there, I can't make the assumption about the naming of the various variables in use nor that the first argument in the array is declaring the sub-command name with the help text attached, so much of what makes it "magic" here would go away. It would just be a `parseArgs` function that does the `sizeof(args) / sizeof(Arg *)` trick for you, and not much else.